SplunkAppForWazuh/default/app.conf

```diff
@@ -3,7 +3,7 @@ is_visible = 1
 label = Wazuh
 
 [launcher]
-version = 4.0.4
+version = 4.1.3
```

Comment:
I think these changes should not be included in your pr

Reply:
Changed version!